<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoClass</class>
 <widget class="QMainWindow" name="videoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>950</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>神秘快播</string>
  </property>
  <property name="windowIcon">
   <iconset resource="video.qrc">
    <normaloff>:/images/logo.ico</normaloff>:/images/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/images/background.png);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>146</width>
             <height>190</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>146</width>
             <height>190</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QListWidget" name="listWidget_2"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QListWidget" name="listWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>230</height>
            </size>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>190</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>190</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTextBrowser" name="textBrowser">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="video.qrc">
           <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="video.qrc">
           <normaloff>:/images/volumeOpen.png</normaloff>:/images/volumeOpen.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuwenj">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>最近文件</string>
     </property>
     <property name="icon">
      <iconset resource="video.qrc">
       <normaloff>:/images/fileList.png</normaloff>:/images/fileList.png</iconset>
     </property>
     <addaction name="action_2"/>
    </widget>
    <addaction name="openAction"/>
    <addaction name="menu_5"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>字幕</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>在线匹配</string>
     </property>
     <property name="icon">
      <iconset resource="video.qrc">
       <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
     </property>
     <addaction name="searchAction"/>
     <addaction name="autoDownAction"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>字幕时间</string>
     </property>
     <property name="icon">
      <iconset resource="video.qrc">
       <normaloff>:/images/time.png</normaloff>:/images/time.png</iconset>
     </property>
     <addaction name="addAction"/>
     <addaction name="lowerAction"/>
     <addaction name="timeAction"/>
    </widget>
    <addaction name="openAction_2"/>
    <addaction name="showAction"/>
    <addaction name="menu_4"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>目录</string>
    </property>
    <addaction name="listAction"/>
   </widget>
   <addaction name="menuwenj"/>
   <addaction name="menu"/>
   <addaction name="menu_6"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAction">
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/openFile.png</normaloff>:/images/openFile.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="openAction_2">
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/openFile.png</normaloff>:/images/openFile.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="showAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
   </property>
   <property name="text">
    <string>显示字幕</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="searchAction">
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </action>
  <action name="autoDownAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="text">
    <string>自动匹配</string>
   </property>
  </action>
  <action name="addAction">
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/addition.png</normaloff>:/images/addition.png</iconset>
   </property>
   <property name="text">
    <string>提前100ms</string>
   </property>
   <property name="toolTip">
    <string>提前100ms</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="lowerAction">
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/decrease.png</normaloff>:/images/decrease.png</iconset>
   </property>
   <property name="text">
    <string>延后100ms</string>
   </property>
   <property name="toolTip">
    <string>延后100ms</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="timeAction">
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/time.png</normaloff>:/images/time.png</iconset>
   </property>
   <property name="text">
    <string>当前0ms</string>
   </property>
  </action>
  <action name="action_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/not.png</normaloff>:/images/not.png</iconset>
   </property>
   <property name="text">
    <string>空</string>
   </property>
  </action>
  <action name="listAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="video.qrc">
     <normaloff>:/images/fileList.png</normaloff>:/images/fileList.png</iconset>
   </property>
   <property name="text">
    <string>开启目录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="video.qrc"/>
 </resources>
 <connections/>
</ui>
